modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-the-olm-installer.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="disabling-readiness-init-containers-for-the-olm-installer_{context}"]
+= Disabling the readiness init containers for the OLM installer
+
+To disable the previously enabled readiness init containers for {prod-short} installed by the OLM installer:
+
+.Prerequisites
+
+* {prod-short} is installed by the OLM installer.
+
+.Procedure
+
+. Find the {prod-short} Operator subscription name:
```

Comment:
```suggestion
. Get the {prod-short} Operator subscription name:
```

Reply:
To my mind, word `find` is better here as user has to find subscription that corresponds to Che. In my opinion, word `get` is used when we get exact resource by a command.

Reply:
I understand. In such case, please check if this is also the case in the next step, where I already swapped `find` with `get`.

Reply:
Checked, but it was correct